schema.json

```diff
@@ -279,6 +279,7 @@
         "type": ["object", "null"],
         "properties": {
           "driver": {"type": "string"},
+          "name":{"type":"string"},
```

Comment:
Whitespace is missing here, please complete your commit with it.

Reply:
done :)

Reply:
Thank you 👍🏻